tests/cases/fourslash/documentHighlightInExport1.ts

```diff
@@ -0,0 +1,11 @@
+﻿/// <reference path='fourslash.ts'/>

```

Comment:
This doesn't seem document-highlights specific, so would probably be better as `findAllRefsReExports3.ts`. Then you could use `verify.noReferences` instead of adding a `noDocumentHighlights` method.

Reply:
I wrote a FAR test to confirm that it covers this change.  I don't mind submitting it as well, but I think it's worthwhile to keep this test because (1) the scenario does not appear to have other test coverage, (2) it seems like there ought to be at least one test asserting that certain points in the code *don't* have reference highlighting.